### PR TITLE
Azure - new card design

### PR DESCRIPTION
**Pull request by jasmingoh**
Thanks to @xtinah-w for all the css!

We've redesigned the Azure page to include Azure for students!

Here's the mock:
image

### CODE REVIEW

**Review by streetcoder123 — COMMENTED** [review]
Should we remove `introToAzure` svg since I think we are not using it anymore?

**Review by SahilTara — APPROVED** [review]

File: src/client/src/containers/AzureLogin/index.tsx [hunk 1]
```diff
@@ -74,48 +76,56 @@ class AzureLogin extends React.Component<Props> {
...
-                    defaultMessage="Sign out"
-                  />
-                </div>
-              </div>
-            )}
-            {!isLoggedIn && (
+          </div>
+        </div>
+        <AzureStudent />
+        {/* Azure Services Card*/}
```
Comment (jsondoo) [inline src/client/src/containers/AzureLogin/index.tsx:90]: NIT: i think i typed this, but should be a space after Card lol

File: src/client/src/assets/cloudservices.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...oints="101.13 218.275 171.02 218.275 171.691 213.527 101.8 213.527 101.13 218.275"/><polygon class="cls-11" points="99.864 227.242 140.753 227.242 141.423 222.495 100.535 222.495 99.864 227.242"/><path class="cls-21" d="M82.78,216.5522a2.12,2.12,0,0,0,2.0315-1.78,1.51,1.51,0,0,0-1.5289-1.78,2.12,2.12,0,0,0-2.0315,1.78A1.51,1.51,0,0,0,82.78,216.5522Z"/><polygon class="cls-10" points="389.826 269.016 375.465 269.018 312.77 188.315 327.131 188.314 389.826 269.016"/><rect class="cls-11" x="367.0755" y="154.5076" width="9.3987" height="116.6666" transform="translate(-42.1935 102.623) rotate(-14.8821)"/><rect class="cls-13" x="376.4451" y="190.2658" width="4.6994" height="79.6847" transform="translate(-46.3928 105.0052) rotate(-14.8821)"/><rect class="cls-13" x="353.7456" y="154.1206" width="5.4826" height="2.3837" transform="translate(-27.9312 96.7669) rotate(-14.8821)"/><path class="cls-11" d="M391.2984,268.0106l.862,13.7162a2.0493,2.0493,0,0,1-3.8845,1.0323l-6.0609-12.3346Z"/><rect class="cls-1" x="305.9056" y="281.9245" width="86.2595" height="3.0755" rx="1.5378"/><path class="cls-22" d="M387.4864,281.1523l.7895,1.6068a2.0493,2.0493,0,0,0,3.8845-1.0323l-.1123-1.7867Z"/></g></g></svg>
```
Comment (jsondoo) [inline src/client/src/assets/cloudservices.svg:1]: needs alt tag, and role=img

Comment (jsondoo) [reply src/client/src/assets/cloudservices.svg:1]: same with other svg

**Review by xtinah-w — APPROVED** [review]
Looks great to me!

**Review by xtinah-w — COMMENTED** [review]
My bad, I didn't mean to approve this PR, I will review later then update

**Review by streetcoder123 — CHANGES_REQUESTED** [review]
Currently, `sign in` is not triggered from the `add resource` button. Waiting for changes for that to be fixed

**Pull request closed without merging**